akka-docs/rst/java/lambda-persistence.rst

```diff
@@ -202,6 +190,46 @@ and before any other received messages.
 If there is a problem with recovering the state of the actor from the journal, ``onRecoveryFailure``
 is called (logging the error by default), and the actor will be stopped.
 
+.. _internal-stash-lambda:
+
+Internal stash 
+--------------
+
+The persistent actor has a private :ref:`stash <stash-lambda>` for internally caching incoming messages during 
+:ref:`recovery <recovery-java-lambda>` or the ``persist\persistAll`` method persisting events. However You can use 
+inherited stash. The internal stash doesn't interfere with it apart from inherited ``unstashAll`` method, which 
```

Comment:
"you can still use/inherit from the Stash interface"
"The internal stash cooperates with the normal Stash by hooking into unstashAll and making sure messages are unstashed properly to the internal stash to maintain ordering guarantees."